MediaBrowser.Model/Tasks/ITaskManager.cs

```diff
@@ -17,7 +17,7 @@ public interface ITaskManager : IDisposable
         /// Gets the list of Scheduled Tasks.
         /// </summary>
         /// <value>The scheduled tasks.</value>
-        IScheduledTaskWorker[] ScheduledTasks { get; }
+        IReadOnlyCollection<IScheduledTaskWorker> ScheduledTasks { get; }
```

Comment:
Any reason to use `IReadOnlyCollection` over `IReadOnlyList`?

Reply:
no particular